controller/src/main/java/org/jboss/as/controller/management/BaseHttpInterfaceResourceDefinition.java

```diff
@@ -181,8 +182,37 @@ public void validateParameter(String parameterName, ModelNode value) throws Oper
             .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
             .build();
 
+    public static final SimpleAttributeDefinition BACKLOG = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.BACKLOG, ModelType.INT, true)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(50))
+            .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
+            .setStability(Stability.COMMUNITY)
+            .build();
+
+    public static final SimpleAttributeDefinition NO_REQUEST_TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.NO_REQUEST_TIMEOUT, ModelType.INT, true)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(60000))
```

Comment:
Please add a setMeasurementUnit

Reply:
This is added, BTW @bstansberry GitHub says FAHRENHEIGHT was your contribution? :smile:

Reply:
Somebody hacked me!

Reply:
Not guilty! Except of careless cutting and pasting!

https://access.redhat.com/webassets/avalon/d/Red_Hat_JBoss_Operations_Network-3.1-API-en-US/files/domain/index.html?org/rhq/core/domain/measurement/MeasurementUnits.html